docs/quickstart_guide.md

```diff
@@ -6,15 +6,17 @@ In this guide, you will create and run a Flyte workflow in a local Python enviro
 ````{dropdown} Try Flyte in your browser
 :animate: fade-in-slide-down
 
-```{button-link} https://sandbox.union.ai/
+Union Serverless is a hosted version of Flyte that you can try in your browser. To get started, see the [Union Serverless Quickstart guide](https://docs.union.ai/serverless/quick-start).
+
+```{button-link} https://serverless.union.ai/
```

Comment:
We should give link to the signup instead of the direct url to serverless as we are still gating access to this. Signup.union.ai is where they join the waitlist and once approved they get mail with instructions on accessing serverless which includes that url